plugins/tracker-resources/src/components/issues/related/RelatedIssues.svelte

```diff
@@ -65,18 +69,25 @@
   )
 </script>
 
-<div class="mt-1">
-  {#if subIssues !== undefined && viewlet !== undefined}
-    {#if issueStatuses.size > 0 && teams}
-      <SubIssueList bind:viewOptions {viewlet} issues={subIssues} {teams} {issueStatuses} />
-    {:else}
-      <div class="p-1">
-        <Label label={presentation.string.NoMatchesFound} />
-      </div>
-    {/if}
+{#if subIssues !== undefined && viewlet !== undefined}
+  {#if issueStatuses.size > 0 && teams && subIssues.length > 0}
+    <SubIssueList bind:viewOptions {viewlet} issues={subIssues} {teams} {issueStatuses} disableHeader />
```

Comment:
I'm disagree to disable header always, we need to do it only in case no grouping are used.